test/eyeshade/ads.integration.test.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+import {
+  serial as test
+} from 'ava'
+import uuidV4 from 'uuid/v4'
+import {
+  cleanPgDb,
+  connectToDb,
+  cleanDbs,
+  dbUri
+} from '../utils'
+import Postgres from 'bat-utils/lib/runtime-postgres'
+import { monthly } from '../../eyeshade/workers/ads'
+import { Runtime } from 'bat-utils'
+
+const postgres = new Postgres({ postgres: { url: process.env.BAT_POSTGRES_URL } })
+const runtime = new Runtime({
+  postgres: { url: process.env.BAT_POSTGRES_URL },
+  database: {
+    mongo: dbUri('ledger')
+  }
+})
+
+test.afterEach.always(async t => {
+  await cleanPgDb(postgres)()
+  await cleanDbs()
+})
+
+test('ads payout report cron job takes a snapshot of balances', async t => {
+  const eyeshadeMongo = await connectToDb('ledger')
+  const wallets = eyeshadeMongo.collection('wallets')
```

Comment:
can we just use the runtime db here?

Reply:
Probably, I will check and revert.  Also need to update the variable name since it's no longer eyeshadeMongo